graphene_django/relay/connection.py

```diff
@@ -305,3 +294,18 @@ def _handle_last_before(
         has_previous_page,
         has_next_page,
     )
+
+
+def _use_connection_dataloaders(info, sized_sliceable):
```

Comment:
Curious to know why we've reverted back to a succession of progressive `hasattr`.

Wasn't the 

```py
try:
    dataloaders_context = info.context.dataloaders
except AttributeError:
    pass
```

+ later checks more optimal?

Reply:
Added suggestion below

Reply:
applied, thanks